community-operators/percona-server-mongodb-operator/1.7.0/percona-server-mongodb-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,399 @@
+    statically assigned IP addresses.
+
+
+    * **Utilize Local Storage Options** - [with support for Local Storage you
+    can mount existing data
+    directories](https://www.percona.com/doc/kubernetes-operator-for-psmongodb/storage.html)
+    into your replica set managed by Kubernetes or utilize high performance
+    hardware for local storage rather than network storage for your database.
+
+
+    ### Before You Start
+
+
+    Add the PSMDB user `Secret` to Kubernetes. User information must be placed
+    in the data section of the `secrets.yaml`
+
+    file with Base64-encoded logins and passwords for the user accounts.
+
+
+    Below is a sample `secrets.yaml` file for the correct formatting.
+
+
+    ```
+
+    apiVersion: v1
+
+    kind: Secret
+
+    metadata:
+      name: my-cluster-name-secrets
+    type: Opaque
+
+    data:
+      MONGODB_BACKUP_USER: YmFja3Vw
+      MONGODB_BACKUP_PASSWORD: YmFja3VwMTIzNDU2
+      MONGODB_CLUSTER_ADMIN_USER: Y2x1c3RlckFkbWlu
+      MONGODB_CLUSTER_ADMIN_PASSWORD: Y2x1c3RlckFkbWluMTIzNDU2
+      MONGODB_CLUSTER_MONITOR_USER: Y2x1c3Rlck1vbml0b3I=
+      MONGODB_CLUSTER_MONITOR_PASSWORD: Y2x1c3Rlck1vbml0b3IxMjM0NTY=
+      MONGODB_USER_ADMIN_USER: dXNlckFkbWlu
+      MONGODB_USER_ADMIN_PASSWORD: dXNlckFkbWluMTIzNDU2
+      PMM_SERVER_USER: cG1t
+      PMM_SERVER_PASSWORD: c3VwYXxefHBheno=
+    ```
+
+
+    * Add support for sharding to scale MongoDB cluster horizontally
+
+    * Support for custom sidecar container to extend the Operator capabilities
+
+    * Persistent Volume Claims can now be automatically removed after MongoDB cluster deletion
+
+    * Operator can now automatically remove old backups from S3 if retention period is set
+
+    * Add support for runtimeClassName Kubernetes feature for selecting the container runtime
+
+    * It is now possible to explicitly set the version of MongoDB for newly provisioned clusters. Before that, all new clusters were started with the latest MongoDB
+    version if Version Service was enabled
+
+    * Fix confusing log messages about no backup / restore found which were caused by Percona Backup for MongoDB waiting for the backup metadata
+
+    * MongoDB container liveness probe will now use TLS to follow best practices and remove noisy log messages from mongod log
+
+    * Fix a bug which prevented adding/removing labels to Pods without downtime
+
+    * Fix a bug which prevented enabling Percona Monitoring and Management (PMM) due to incorrect request for the recommended PMM Client image version to the Version Service running multiple replica sets without sharding enabled should be prohibited
+
+    * Fix a bug which caused mongos process to fail when using allowUnsafeConfigurations=true
+
+    * Fix a bug due to which changing secrets in a single-shard mode caused mongos Pods to fail
+
+  maturity: stable
+  version: 1.7.0
+  replaces: 'percona-server-mongodb-operator.v1.7.0'
```

Comment:
Please replace it with previous version `  replaces: 'percona-server-mongodb-operator.v1.6.0'`

Reply:
oh, sorry. My bad